### PR TITLE
Added @mozilla/rally-functions

### PR DESCRIPTION
**Pull request by thomik-corp**
Remove functions folder and symlink to @mozilla/rally-functions
Remove functions only test as it is now part of monorepo
Fixes: https://github.com/mozilla-rally/rally/issues/29

### CODE REVIEW

**Comment by aaga** [comment]
`npm run dev` doesn't work out of the box anymore. Either we should add pre-build inside run dev, or just add the link functions step directly.

**Review by aaga — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -2,29 +2,25 @@
   "name": "rally-web-platform",
   "version": "0.1.3",
   "scripts": {
-    "pre-build": "bash scripts/reset-build-directories.sh && bash scripts/add-protocol.sh && node scripts/add-version-object.js",
-    "dev": "npm run config:web:demo && npm run build:functions && firebase emulators:exec --project demo-rally --only auth,functions,firestore --ui 'npm run load:data && (npm run watch:functions & npm run watch:web)'",
+    "pre-build": "bash scripts/reset-build-directories.sh && bash scripts/add-protocol.sh && node scripts/add-version-object.js && ln -fs node_modules/@mozilla/rally-functions functions",
+    "dev": "npm run config:web:demo && firebase emulators:exec --project demo-rally --only auth,functions,firestore --ui 'npm run load:data && npm run watch:web'",
```
Comment (aaga) [inline package.json:6]: `npm run dev` doesn't work out of the box anymore. Either we should add pre-build inside run dev, or just add the link functions step directly.

**Review by aaga — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -137,6 +137,8 @@ static/firebase.config.json
 *.swo
 *~
 #Local Netlify folder
-.netlify 
+.netlify
 # Local Netlify folder
 .netlify
```
[the remark this answers is not in the document]
Comment (thomik-corp) [reply .gitignore:142]: Done

File: package.json (hunk 1 above)
Comment (thomik-corp) [reply package.json]: Had to symlink functions here. It works now. Thanks

**Pull request merged**